src/ui/app/App.js

```diff
@@ -191,7 +191,7 @@ const appsRoutes = entandoApps.reduce((routes, app) => (
         key={AppRoute.path}
         path={AppRoute.path}
         // eslint-disable-next-line react/jsx-pascal-case
-        render={() => <InternalPage><AppRoute.component /></InternalPage>}
+        render={() => <PageLayout><AppRoute.component /></PageLayout>}
```

Comment:
here you don't need to wrap AppRoute.component with PageLayout because is already inherited inside the component App.

Reply:
done ✅